test/check-machines-consoles

```diff
@@ -744,6 +745,102 @@ fullscreen=0
         b.wait_visible(".vm-console-vnc canvas")
         self.assertIn("password=on", m.execute(f"grep ^-vnc /var/log/libvirt/qemu/{name}.log"))
 
+        # Change global password, but without restarting the VM. This
+        # means that Cockpit will use the new password when
+        # connecting, but the VM still expects the old.
+        m.execute("sed -i -e 's/vnc_password = \"barfoo\"/vnc_password = \"foobar\"/' /etc/libvirt/qemu.conf")
+
+        # We are still connected, but reconnecting will fail.
+
+        b.click(".consoles-card button:contains(Disconnect)")
+        b.wait_in_text(".consoles-card", "Disconnected")
+
+        # Quite often the reconnection here fails with code 1006
+        # (Abnormal Closure), but retrying will give us the expected
+        # error message, usually on the second try.
+
+        b.click(".consoles-card button:contains(Connect)")
+        for _ in range(5):
+            try:
+                b.wait_in_text(".consoles-card", "Authentication failed")
```

Comment:
This will take 5 * 30 s = 2.5 minutes at worst. Perhaps reduce the timeout of the wait_in_text? (`with b.wait_timeout(10):` or so)

Reply:
Hmm, isn't the default timeout already 10s?

Reply:
oh right, it's 15. I hope in most cases that will already work on the second iteration? then you can leave it as-is.

Reply:
Ack, good 'nuff.